_docs/index.md

```diff
@@ -22,6 +22,8 @@ is where you can learn about what Istio does and how it does it.
   the Istio control plane in various environments (e.g., Kubernetes, Consul,
   etc.), as well as instructions for installing the sidecar in the
   application deployment.
+  Jump directly to [downloading and installing](https://istio.io/docs/setup/kubernetes/quick-start.html) Istio for Kubernetes,
+  if it is your platform of choice.
```

Comment:
happy to change the english around it as long as there is a directly link to download & install (for k8s)